engine/script/src/test/test_script_table.cpp

```diff
@@ -168,7 +168,7 @@ TEST_F(LuaTableTest, AttemptReadUnsupportedVersion)
     int result = lua_cpcall(L, ReadUnsupportedVersion, 0x0);
     ASSERT_NE(0, result);
     char str[256];
-    dmSnPrintf(str, sizeof(str), "Unsupported serialized table data: version = 0x%x (current = 0x%x)", 818192, 3);
+    dmSnPrintf(str, sizeof(str), "Unsupported serialized table data: version = 0x%x (current = 0x%x)", 818192, 4);
```

Comment:
Does that means we can't read old saves anymore? 
If we can, do we have a test that check this?

Reply:
No, no, we can still read old versions. BUT I don't think I have a test for reading a v3 table with the v4 serializer. I might as well add one!